soc/arm/ambiq/apollo4x/soc.c

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "hal/am_hal_utils.h"
```

Comment:
not a local header, please don't use `""`

Reply:
Removed this include entirely, it is not needed